pkg/trace/stats/concentrator.go

```diff
@@ -165,7 +165,8 @@ func (c *Concentrator) addNow(pt *traceutil.ProcessedTrace, containerID string)
 	}
 	for _, s := range pt.TraceChunk.Spans {
 		isTop := traceutil.HasTopLevel(s)
-		if !(isTop || traceutil.IsMeasured(s)) || traceutil.IsPartialSnapshot(s) {
+		isRemoteOutgoing := traceutil.IsRemoteOutgoing(s)
+		if !(isTop || traceutil.IsMeasured(s) || isRemoteOutgoing) || traceutil.IsPartialSnapshot(s) {
```

Comment:
I think this might be clearer if you separate the negative and positive cases like
```suggestion
		if !(isTop || traceutil.IsMeasured(s) || traceutil.IsRemoteOutgoing(s)) {
			continue
		}
		if traceutil.IsPartialSnapshot(s) {
			continue
		}
```

Reply:
I was thinking the same, but wasn't sure if it was Go style to keep it as one line. I'll make this change, thanks!